google/cloud/aiplatform/docker_utils/build.py

```diff
@@ -174,18 +174,18 @@ def _prepare_entrypoint(package: Package, python_command: str = "python") -> str
             Required. The python command used for running python code.
 
     Returns:
-        A string with Dockerfile directives to set ENTRYPOINT.
+        A string with Dockerfile directives to set ENTRYPOINT or "".
     """
-
+    exec_str = ""
     # Needs to use json so that quotes print as double quotes, not single quotes.
     if package.python_module is not None:
         exec_str = json.dumps([python_command, "-m", package.python_module])
-    else:
+    elif package.script is not None:
         _, ext = os.path.splitext(package.script)
         executable = [python_command] if ext == ".py" else ["/bin/bash"]
         exec_str = json.dumps(executable + [package.script])
 
-    return "\nENTRYPOINT {}\n".format(exec_str)
+    return "\nENTRYPOINT {}\n".format(exec_str) if exec_str else exec_str
```

Comment:
I think in this case, it's clearer to write as:

```
if not exec_str:
  return ""
return "\nENTRYPOINT {}\n".format(exec_str)

Reply:
Done.